packages/plugin-core/src/KeybindingUtils.ts

```diff
@@ -1,206 +1,322 @@
+      `1. Open [keybindings.json](${keybindingJSONCommandUri})`,
+      `1. Paste the clipboard content to \`keybindings.json\``,
+      "",
+      "#### 2. Remap conflicting keybindings",
+      "If you need to preserve the keybinding that is conflicting, consider remapping either of the conflicting keybindings.",
+      `1. Click on the link \`Remap\` next to each conflicting keybinding listed below.`,
+      `    - This will copy the necessary keybinding entry to your clipboard.`,
+      `1. Open [keybindings.json](${keybindingJSONCommandUri})`,
+      `1. Paste the clipboard content to \`keybindings.json\``,
+      `1. Set the value of "key" to the desired key combination.`,
+      "",
+      "For more information on how to set keyboard rules in VSCode, visit [Keyboard Rules](https://code.visualstudio.com/docs/getstarted/keybindings#_keyboard-rules)",
+      "",
+      `Use [Default Keybindings](${defaultKeybindingJSONCommandUri}) to reference all default keybindings.`,
+      "",
+      "***",
+      "## List of Keybinding Conflicts",
+      conflicts
+        .map((conflict) => {
+          const conflictKeybindingEntry = defaultKeybindingJSON.find(
+            (keybinding) => {
+              return keybinding.command === conflict.commandId;
+            }
+          );
+          const dendronKeybindingEntry = defaultKeybindingJSON.find(
+            (keybinding) => {
+              return keybinding.command === conflict.conflictsWith;
+            }
+          );
+          if (conflictKeybindingEntry === undefined) {
+            return undefined;
+          }
+          if (dendronKeybindingEntry === undefined) {
+            return undefined;
+          }
 
-    // add override if there isn't.
-    const OVERRIDE_EXPAND_LINE_SELECTION = {
-      key: `ctrl+l`,
-      command: "-extension.vim_navigateCtrlL",
-    };
+          const disableBlock = KeybindingUtils.generateKeybindingBlockForCopy({
+            entry: conflictKeybindingEntry,
+            disable: true,
+          });
+
+          const remapDendronBlock =
+            KeybindingUtils.generateKeybindingBlockForCopy({
+              entry: dendronKeybindingEntry,
+              disable: false,
+            });
 
-    const newKeybindings = assign(
-      keybindings,
-      keybindings.concat(OVERRIDE_EXPAND_LINE_SELECTION)
+          const remapConflictBlock =
+            KeybindingUtils.generateKeybindingBlockForCopy({
+              entry: conflictKeybindingEntry,
+              disable: false,
+            });
+
+          const copyCommandUri = (args: { text: string }) =>
+            `command:dendron.copyToClipboard?${encodeURIComponent(
+              JSON.stringify(args)
+            )}`;
+
+          const out = [
+            `### \`${conflict.commandId}\``,
+            `- key: \`${conflictKeybindingEntry.key}\``,
+            `- command: \`${conflict.commandId}\` [Disable](${copyCommandUri({
```

Comment:
nit: maybe expand the description here.  Right now it reads like

```
[Disable extension.vim_tab]
[Remap extension.vim_tab]
[Remap dendron.lookupNoteAutoComplete]
```

But maybe expand to something like 

```
Disable extension.vim_tab - Click to generate JSON.
...
```

So to avoid the confusion that a user may think clicking on the link is enough to fix the issue (in case they didn't properly read the instructions, like myself the first time. 😄 

And maybe in the pop-up that comes up, we can expand it from 'text copied to clipboard' to something like 'text copied, now add to your keybindings.json', and maybe have a button that takes them there as a cherry-on-top.  These are just nits though.

Reply:
> maybe expand the description here

I went the other way of condensing it and sticking them next to the command ids to make it less busy 😅
Any ideas on how to make it more explicit  and also not too crowded? It's a bit hard to format these with markdown.

> maybe have a button that takes them there as a cherry-on-top.

I could make `CopyToClipboard` accept callbacks to make this work (or just make a separate command), but not sure if we can pass the callback after encoding it as a URI component. I think having a link in the webview like right now would be enough for visibility.

Reply:
As I was saying - mostly a nit.  I think your less busy approach is also fine.